lib/peeps.rb

```diff
@@ -0,0 +1,24 @@
+class Peeps
+
+  def self.all
+    query = "SELECT * FROM peeps ORDER BY created;"
+    DBWrapper.database_send(query).map { |row| initialize(row) }   
```

Comment:
if you add DESC to the end of the query on row 4 it will put them in descending chronological order :)